packages/local_auth/android/src/main/java/io/flutter/plugins/localauth/LocalAuthPlugin.java

```diff
@@ -59,7 +60,7 @@ public void onMethodCall(MethodCall call, final Result result) {
             null);
```

Comment:
There's a bug here for `authInProgress`. We might set it to true and never actually call `authenticate`.

Could you chage line 41 to: `if (authInProgress.get())`
then add to line 62: `authInProgress.set(true);`.

Since this method always runs in the `UIThread`, we technically don't need to use AtomicBoolean. You can also optionally change all of this to a normal `boolean` primitive.

Reply:
Resolved.

Reply:
Resolved. Not changing it to boolean primitive. Since, I don't want things to break. :)